scripts/xtensa-build-zephyr.sh

```diff
@@ -89,11 +99,16 @@ build()
 	local STAGING=build-sof-staging
 	mkdir -p ${STAGING}/sof/ # smex does not use 'install -D'
 
+        # Default RIMAGE_KEY defined at the top of this file
 	for platform in "${PLATFORMS[@]}"; do
 		case "$platform" in
 			apl)
 				PLAT_CONFIG='intel_adsp_cavs15'
 				;;
+			imx)
+				PLAT_CONFIG='intel_adsp_cavs15'
```

Comment:
Hasty copy/paste... @iuliana-prodan which `--board` parameter have you been using?

Reply:
I believe it's nxp_adsp_imx8.

Reply:
yes, is ` PLAT_CONFIG='nxp_adsp_imx8'`